packages/pubsub/src/connection-pool.js

```diff
@@ -32,6 +32,9 @@ var uuid = require('uuid');
 var PKG = require('../package.json');
 var v1 = require('./v1');
 
+var CHANNEL_READY = 'channel.ready';
```

Comment:
Can we long-form name these, i.e. `CHANNEL_READY_EVENT_NAME`

Reply:
Sure!